Userland/Libraries/LibCore/Process.cpp

```diff
@@ -160,9 +162,37 @@ ErrorOr<bool> Process::is_being_debugged()
         return (tracer_pid != 0UL);
     }
     return false;
+#elif defined(AK_OS_MACOS)
+    // https://developer.apple.com/library/archive/qa/qa1361/_index.html
+    int junk;
+    int mib[4];
+    struct kinfo_proc info;
+    size_t size;
```

Comment:
can we initialize/zero initialize these variables as soon as they are declared? 

```
int mib[4] = {};
struct kinfo_proc info = {};
size_t size = sizeof(info);


mib[0] = ...;
...
```

Then you won't need to  have a note about initializing the p_flag field.